src/hentai.rs

```diff
@@ -218,6 +219,44 @@ impl Hentai
             }
             if image_download_success {break;} // if all images were downloaded successfully: continue with cbz creation
         }
+        if !image_download_success && webarchive == true { // Web Archive Loop
```

Comment:
I don't want to download images from web archive, only the metadata as the images should not have been purged from the nhentai media servers (see famously 177013, last section in my readme). Correct me if I'm wrong.

Then hypothetically speaking, if we wanted to download images from the web archive: Completely duplicating the outer download logic is likely unnecessary. If I see this correctly, the only thing that's different is that you call a different download image function `Self::archive_image`. Why didn't you just implement fallback to web archive logic in the original `download_image` function?